gstreamer-rtsp/src/auto/r_t_s_p_watch_funcs.rs

```diff
@@ -0,0 +1,8 @@
+// This file was generated by gir (0113735) from gir-files (???)
+// DO NOT EDIT
+
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use std::mem;
+use std::ptr;
```

Comment:
Lots of empty files here. Just disable the relevant API for now

Reply:
done